lib/src/expanded/subscription_expanded.dart

```diff
@@ -1,48 +1,18 @@
 import 'package:stripe/messages.dart';
 import 'package:stripe/src/expanded.dart';
-import 'package:stripe/src/utils/expandable_fields/customer_expandable_field.dart';
-import 'package:stripe/src/utils/expandable_fields/discounts_expandable_field.dart';
-import 'package:stripe/src/utils/expandable_fields/latest_invoice_expanded_expandable_field.dart';
+import 'package:stripe/src/expanded/customer_expanded.dart';
+import 'package:stripe/src/expanded/discount_expanded.dart';
 
 class SubscriptionExpanded {
   final Subscription subscription;
-  final List<Discount>? discounts;
+  final List<DiscountExpanded>? discounts;
   final InvoiceExpanded? latestInvoice;
-  final Customer? customer;
+  final CustomerExpanded? customer;
```

Comment:
This is probably the pattern we need to stick with. In most cases the included fields will have nesting of their own, and in order to minimize breaking changes as we add more "expanded" fields -- we'll need to make all nested fields `xExpanded`:

```dart
class ObjExpanded {
  final Obj obj;
  final NestedAExpanded nestedA;
  final NestedB...
}
```